### PR TITLE
feat(pages): move static cf pages to frontend

### LINKED ISSUE #288: Move legal pages to frontend
Currently our legal pages are rendered by the CF-Worker. These are the sites:

* https://de.serlo.org/imprint
* https://de.serlo.org/privacy
* https://de.serlo.org/terms

It would be great when we move those sites to the frontend. The rendering is done by the module https://github.com/serlo/serlo.org-cloudflare-worker/tree/main/src/legal-pages Refactoring of the source code there would be also nice :-)

### PR DESCRIPTION
**Pull request by elbotho**
- [404 for meeting redirect](https://frontend-git-cf-worker-static-stuff-serlo.vercel.app/___meeting_not_found)
- Impressum ([DE](https://frontend-git-cf-worker-static-stuff-serlo.vercel.app/legal),[EN](https://frontend-git-cf-worker-static-stuff-serlo.vercel.app/en/legal))
- Terms ([DE](https://frontend-git-cf-worker-static-stuff-serlo.vercel.app/terms),[EN](https://frontend-git-cf-worker-static-stuff-serlo.vercel.app/en/terms))
- Privacy ([DE](https://frontend-git-cf-worker-static-stuff-serlo.vercel.app/privacy),[EN](https://frontend-git-cf-worker-static-stuff-serlo.vercel.app/en/privacy))

This is a bit quick and dirty, but since we almost never change it ...

I simplified the privacy part a bit.
It now fetches the commit date from the GitHub-API and the archive just links to the markdown files on GitHub(good enough, no?) so we don't have to manually change a list

### CODE REVIEW

File: src/pages/___meeting_not_found.tsx
Comment (kulla) [inline src/pages/___meeting_not_found.tsx]: Side note: A generic `not_found` page would be enough for the CF worker.

File: src/pages/legal.tsx
Comment (kulla) [inline src/pages/legal.tsx]: Do we actually need this setting?

File: src/pages/privacy.tsx
Comment (kulla) [inline src/pages/privacy.tsx:38]: Shall we add this to our `i18n`?

File: src/pages/privacy.tsx
Comment (kulla) [inline src/pages/privacy.tsx:88]: Here you can use `io-ts` to make it more secure...

File: src/pages/privacy.tsx
Comment (kulla) [inline src/pages/privacy.tsx]: This seems to be duplicated code which can be stored in a utils function

**Review by kulla — APPROVED** [review]
Thank you. Much better and simpler than the current implementation! :heart: I have added some notes. However nothing which blocks merging the PR. Only the `allowDangerousHtml` can be checked again before merging whether this is really needed.

File: src/pages/privacy.tsx
Comment (elbotho) [reply src/pages/privacy.tsx:38]: I opted to leave it out for now since we seldomly change it and we'll probably only have `de` and `en` versions. But not a strong opinion here.

File: src/pages/legal.tsx
Comment (elbotho) [reply src/pages/legal.tsx]: input markdown example:
```
Serlo Education e. V.<br>
IBAN: DE98 4306 0967 8204 5906 00<br>
BIC: GENODEM1GLS (GLS Bank)

Knr: 8204590600<br>
BLZ: 43060967
```

If we don't use the setting we end up with `<br>` in the output.
Options:
- make sure we never use any html in the markdown files
- add the [tagfilter plugin](https://github.com/micromark/micromark-extension-gfm-tagfilter)
- leave it as it is since the markdown is not user generated and the [source repo](https://github.com/serlo/serlo.org-legal/blob/main/de/imprint.md?plain=1) is as secure (or more so) as this repo

Comment (kulla) [reply src/pages/legal.tsx]: Or we add a replace all for `&lt;br&gt;` to `<br>`...

File: src/pages/privacy.tsx
Comment (elbotho) [reply src/pages/privacy.tsx:88]: secure in what way?
If it's **not** the expected result the frontend will not build, so I'd say it's pretty secure ;)

File: src/pages/legal.tsx
Comment (elbotho) [reply src/pages/legal.tsx]: nice idea

File: src/pages/privacy.tsx
Comment (elbotho) [reply src/pages/privacy.tsx]: now with the manual html conversion I agree it's worth it :)

File: src/pages/___meeting_not_found.tsx
Comment (elbotho) [reply src/pages/___meeting_not_found.tsx]: okay. I made it more general and added a status code 404. 
After we de https://github.com/serlo/backlog/issues/287 we could actually just use the default 404 as well.

**Comment by elbotho** [comment]
@kulla thank you for you feedback, implemented most of it and did some other improvements.
Maybe you could give it another quick review.

- https://github.com/serlo/frontend-e2e-tests/pull/79
- https://github.com/serlo/cloudflare-worker/pull/590

**Review by kulla — APPROVED** [review]
Thank you! :heart:

File: src/pages/privacy.tsx
Comment (kulla) [reply src/pages/privacy.tsx:88]: > If it's not the expected result the frontend will not build, so I'd say it's pretty secure ;)

Secure in a sense that not an `undefined` error is thrown but the can return a good error code (and have a sentry reporting)

Comment (elbotho) [reply src/pages/privacy.tsx:88]: > and have a sentry reporting

frontend would not build and report an error building (including the path, e.g. `/legal`) so we do not need sentry here imho.

(these are completely static sites, they will not be revalidated after build)

**Comment by kulla** [comment]
Merging it, so that https://github.com/serlo/cloudflare-worker/pull/590 can be merged and tested...

**Pull request merged**
